README.md

```diff
@@ -59,15 +45,61 @@ to use Kubernetes as a PaaS (Platform as a Service).
 
 ## Documentation
 
-Installation and user documentation is available at our main [docs.epinio.io](https://docs.epinio.io/) site.
+Detailed installation and user documentation is available at our main
+[docs.epinio.io](https://docs.epinio.io/) site.
+
+### Installation
+
+The basic boilerplate requires a Kubernetes cluster, an Ingress Controller and a Cert Manager as
+detailed in the [documentation](https://docs.epinio.io/installation/install_epinio). Once this is in
+place, and leaving out DNS setup, in the most trivial case the main installation boils down to
+
+```
+helm repo add epinio https://epinio.github.io/helm-charts
+helm repo update
+
+helm install --namespace epinio --create-namespace epinio epinio/epinio \                                               --set global.domain=mydomain.example.com
+```
+
+For the details glossed over here see the
+[documentation](https://docs.epinio.io/installation/install_epinio).
+
```

Comment:
Do we want to mention the possibility of installing Epinio from Rancher? (with possible link to [Rancher installation](https://docs.epinio.io/howtos/install_epinio_on_rancher) or not needed?

Reply:
I believe that just driving readers to the full docs will be enough. The installation docs are focused on Rancher.